cdxev/__main__.py

```diff
@@ -709,6 +711,40 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of a SBOM for manual completion"),
+    )
+    parser.add_argument(
+        "--name",
+        metavar="<schema path>",
+        help=("Name of the software the SBOM belongs to"),
+        default=None,
+        type=str,
+    )
+    parser.add_argument(
+        "--version",
+        metavar="<schema path>",
+        help=("Version of the software the SBOM belongs to"),
+        default=None,
+        type=str,
+    )
+    parser.add_argument(
+        "--supplier",
+        metavar="<schema path>",
+        help=("Name of the supplier of the software the SBOM belongs to"),
```

Comment:
Is this really correct help text? You are also using it for the `metadata.supplier` who is the supplier of the SBOM, respectively information within the SBOM, not necessarily the supplier of the Software. Such an assumption should be noted here as it's also used for the creation of the documentation.

Reply:
Should i use two variables, there might be a use case, where you create an SBOM for a Software you are not supplying?

Reply:
done